pix4d-dependabot/lib/helpers/path_level.rb

```diff
@@ -8,6 +8,15 @@ def select_path_per_directory(git_tree, directory)
   files.map { |item| item["path"] }
 end
 
+def append_to_path(path)
+  return path if (path.length == 1) && (path == "/")
```

Comment:
This would do the same, wouldn't it?
```suggestion
  return path if path == "/"
```

Reply:
Done as suggested.